lib/build.gradle

```
@@ -1,3 +1,6 @@
+plugins {
+    id("com.google.devtools.ksp").version("1.7.0-1.0.6")
```

Nit: Pretty sure the more Groovy way to declare is like so:
```suggestion
    id "com.google.devtools.ksp" version "1.7.0-1.0.6"
```

For reference, please see https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block